spec/observables/dom/ajax-spec.ts

```diff
@@ -763,12 +814,11 @@ describe('Observable.ajax', () => {
     const request = MockXMLHttpRequest.mostRecent;
 
     expect(() => {
-      request.onprogress((<any>'onprogress'));
+      request.upload.onprogress((<any>'onprogress'));
```

Comment:
it seems like we should have to test both paths for this. It's tedious, but I want to make sure this works in every browser we can.